### DISCUSSION TITLE
What do do with Windows 11?

### DISCUSSION THREAD

**Discussion by thepwrtank18** [File-New-Project/EarTrumpet#869] [Q&A] [answered]
Since Windows 11 gives a completely different design, what will EarTrumpet do?

**Comment by thepwrtank18** [comment]
You also can't just put the default sound icon in the misc tray and just use the EarTrumpet one anymore. You have to either show both or put the EarTrumpet one in the misc tray and use it when needed.

**Comment by riverar** [comment, marked as answer]
We're currently evaluating Windows 11. The Insider program has only delivered one build, so we can't make any decisions just yet. Stay tuned!

**Reply by jarvis-1805** [reply]
Hoping to see a completely new design!
